packages/bootstrap/extra/dcos_internal_utils/cli.py

```diff
@@ -72,6 +70,7 @@ def noop(b, opts):
     'dcos-history': noop,
     'dcos-mesos-dns': noop,
     'dcos-net': noop,
+    'dcos-telegraf': noop,
```

Comment:
Will we want separate names for the components on agents/masters like we had before? Or just keep the same name and ely on the config to tell us what to do differently?